janusgraph-core/src/main/java/org/janusgraph/graphdb/olap/job/IndexRepairJob.java

```diff
@@ -113,7 +113,7 @@ protected void validateIndexStatus() {
         }
         Preconditions.checkArgument(isValidIndex, "The index %s is in an invalid state and cannot be indexed. %s", indexName, invalidIndexHint);
         // TODO consider retrieving the current Job object and calling killJob() if !isValidIndex -- would be more efficient than throwing an exception on the first pair processed by each mapper
-        log.debug("Index {} is valid for re-indexing");
+        log.debug("Index {} is valid for re-indexing", index.name());
```

Comment:
index.name() will lead to a backend call.

Reply:
Thanks for noticing it!
I can wrap into `if`. Something like:
```
if(log.isDebugEnabled()){
  log.debug("Index "+index.name()+" is valid for re-indexing");
}
```
Do you think that's OK or do you think we should simply remove / change the log message?

Reply:
@li-boxuan I wrapped it into `if` block as commented above